pkg/rook/purge_osd.go

```diff
@@ -51,5 +53,9 @@ func PurgeOsd(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNames
 	}
 	logging.Info("Running purge osd command")
 
-	exec.RunCommandInOperatorPod(ctx, clientsets, cmd, args, operatorNamespace, clusterNamespace, false, true)
+	_, err = exec.RunCommandInOperatorPod(ctx, clientsets, cmd, args, operatorNamespace, clusterNamespace, false)
+	if err != nil {
+		logging.Error(err, "failed to remove osd %s", osdId)
+		os.Exit(1)
+	}
```

Comment:
This block of code is repeated in many places may be a helper function for it helps?

Reply:
Replaced with logging.Fatal() as @subhamkrai suggested.